homepage/index.html

```diff
@@ -0,0 +1,214 @@
+            </div>
+            <div class="real">
+                <img src="/homepage/images/globe.png" alt="">
+                <span>REAL-WORLD SKILLS</span>
+                <p>At access, you will learn to answer the problems by developing solutions for real - time problems in
+                    the
+                    world. Also, you shall have a chance to interact with industry experts as well.</p>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="course-layout">
+        <div class="layout1">
+            <span>Request A Course Layout</span>
+            <p>Go through our Course Layout, and see what we have to offer you. In the Course Layout, we explore:</p>
+            <div class="bullets">
+                <ul>
+                    <li> Curriculum, programming languages, and technology covered</li>
+                    <li> Daily & Weekly Schedule</li>
+                    <li> Fee Structure</li>
+                    <li> Career Support</li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="layout2">
+            <form action="">
+                <input type="text" id="fullname" name="fullname" placeholder="FULL NAME">
+                <input type="tel" id="phone" name="phone" placeholder="PHONE">
+                <input type="email" id="emailid" name="emailid" placeholder="EMAIL ID">
+                <select name="course" id="course" required>
+                    <option value="" disabled selected hidden>COURSE</option>
+                    <option value="ACCESS 101">ACCESS 101</option>
+                    <option value="ACCESS 102">ACCESS 102</option>
+                    <option value="ACCESS 103">ACCESS 103</option>
+                </select>
+            </form>
+            <button>
+                <p>Send me the Layout</p>
+            </button>
+        </div>
+    </div>
+
+    <div class="online">
+        <div class="flex1">
+            <span>Online or Offline?</span>
+            <p>The only difference between our Online and Offline classes is where you’re located. Both options provide:
+            </p>
+            <ul>
+                <li>Live, Real-Time Instruction</li>
+                <li>Scheduled Lectures and Projects</li>
+                <li>Direct access with Mentors</li>
+                <li>Access Study Material</li>
+                <li>Community Networking</li>
+                <li>Career Support</li>
+            </ul>
+            <div class="online-text">
+                <p>If you’d like to know more—call, email, or chat with us.</p>
+            </div>
+            <button>
+                <p>Talk to Admissions</p>
+            </button>
+        </div>
+        <div class="flex2">
+
+            <img src="/homepage/images/Group 328.png" alt="">
+        </div>
+    </div>
+
+
+    <div class="covid">
+        <p>Due to COVID we are only offereing ONLINE Classes</p>
+    </div>
+
+    <div class="apply">
+        <span>How to Apply?</span>
```

Comment:
this entire section is not proper

<img width="489" alt="Screenshot 2021-09-17 at 8 42 34 AM" src="https://user-images.githubusercontent.com/47788183/133718284-9a9e1890-8066-4751-b4d1-2ed607bf96d5.png">